pkgs/servers/mpd/default.nix

```diff
@@ -72,6 +73,7 @@ in stdenv.mkDerivation rec {
     ++ opt mmsSupport libmms
     ++ opt mpg123Support mpg123
     ++ opt aacSupport faad2
+    ++ opt lameSupport lame
```

Comment:
To keep in line with the other lines, I decided to use `lame` instead of `lame.lib`. All of them not using the `.lib` variant, building mpd requires downloading docs and bins for all of these.